java/jshell.support/src/org/netbeans/modules/jshell/editor/CommandCompletionProvider.java

```diff
@@ -337,7 +337,7 @@ protected ImageIcon getIcon() {
                             n = f.getLookup().lookup(Node.class);
                         }
                         if (n != null) {
-                            return (ImageIcon)ImageUtilities.image2Icon(n.getIcon(BeanInfo.ICON_COLOR_16x16));
+                            return new ImageIcon(n.getIcon(BeanInfo.ICON_COLOR_16x16));
```

Comment:
Suspicious. Nobody shall instantiate icons. Code should call into `ImageUtilities` all the time, I believe.

Reply:
It's a case where additional work, including a public API change in CompletionUtilities.renderHtml, would be required to change method signatures from ImageIcon to Icon.

The change made here here is solely for the purpose of avoiding a ClassCastException. Once we actually start adding SVG icons we can go and migrate individual sites from ImageIcon to Icon, as necessary.